mixins/rtl-mixin.md

```diff
@@ -0,0 +1,20 @@
+# RtlMixin
+
+The `RtlMixin` creates `dir` attributes on host elements based on the document's `dir`, enabling components to define RTL styles for elements within their shadow-DOMs via `:host([dir="rtl"])`. It is possible to opt-out our this behavior by explicitly setting a `dir` attribute (ex. for testing).
+
+## Usage
```

Comment:
I've been chilling out on my usage (ha) of "usage" in the docs... mostly because it now seems redundant I guess? Like _of course_ the docs are about how you use the thing. :) I think it made more sense in the non-monorepos because there were various sections on where to find it, how to test it, how to contribute AND how to use it. But those sections are all in the root README now, so these are literally just about how to use it.

Reply:
Hmmm yeah I see.  Some readmes have them, some don't.  I may get a PR together later to go through them.